src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -554,23 +525,25 @@ function IOURequestStepConfirmation({
     );
 }
 
-IOURequestStepConfirmation.propTypes = propTypes;
-IOURequestStepConfirmation.defaultProps = defaultProps;
 IOURequestStepConfirmation.displayName = 'IOURequestStepConfirmation';
 
-export default compose(
-    withCurrentUserPersonalDetails,
-    withWritableReportOrNotFound,
-    withFullTransactionOrNotFound,
-    withOnyx({
-        policy: {
-            key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report ? report.policyID : '0'}`,
-        },
-        policyCategories: {
-            key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${report ? report.policyID : '0'}`,
-        },
-        policyTags: {
-            key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY_TAGS}${report ? report.policyID : '0'}`,
-        },
-    }),
-)(IOURequestStepConfirmation);
+const IOURequestStepConfirmationWithOnyx = withOnyx<IOURequestStepConfirmationProps, IOURequestStepConfirmationOnyxProps>({
+    personalDetails: {
+        key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+    },
```

Comment:
Go ahead and remove personalDetails from here, instead use useCurrentUserPersonalDetails hook, this should do the trick